src/middlewared/middlewared/etc_files/syslog-ng/conf.d/tnaudit.conf.mako

```diff
@@ -2,11 +2,11 @@
     import os
     import textwrap
 
-    AUDITABLE_SERVICES = ['SMB']
-    AUDIT_DIR = '/audit'
+    from middlewared.plugins.audit.utils import AUDITED_SERVICES, audit_file_path, DATASET_PATH
 
     COLUMNS = textwrap.dedent('''
         "aid varchar",
+        "msg_ts INT",
         "vers FLOAT",
```

Comment:
Sorry, but I just noticed this. Float version, why?

Reply:
Because I probably needed to look at it a different way. We already store version in the table name. We can save a column and just delete.